packages/aws-rfdk/lib/deadline/test/render-queue.test.ts

```diff
@@ -1460,7 +1460,7 @@ describe('RenderQueue', () => {
                   'DNSName',
                 ],
               },
-              ':4433" --tls-ca "caarn"\n' +
+              ':4433" --tls-ca "arn:aws:secretsmanager:123456789012:secret:ca/arn"\n' +
```

Comment:
Could we change this to use the `CA_ARN` constant defined above?

Reply:
This has been updated